core/src/apps/solana/layout.py

```diff
@@ -111,14 +113,19 @@ async def confirm_instruction(
 
             account_data: list[tuple[str, str]] = []
             if len(account_value) == 2:
+                token_suffix = ""
+                if account_template.is_token_id and definitions:
+                    token = definitions.get_token(account_value[0])
+                    token_suffix = f" {token.ticker}"
+
                 signer_suffix = ""
                 if account_value[0] == signer_public_key:
                     signer_suffix = f" ({TR.words__signer})"
 
                 account_data.append(
                     (
                         ui_property.display_name,
-                        f"{base58.encode(account_value[0])}{signer_suffix}",
+                        f"{base58.encode(account_value[0])}{signer_suffix}{token_suffix}",
```

Comment:
Please write your self a note/reminder to ask about the UI in the PR to Trezor once we open it. I'm not sure what the best UI here would be but I'd expect that we would also want to display both the token name and ticker here. That's why I'm thinking we might want to have a screen with additional information here (via the "i" button in the top right corner). But that's up to Trezor guys to decide.

(I'm not sure how difficult to implement this would be since we use the `confirm_properties` call to display these data)

Reply:
The answer would IMO then also apply to predefined token transfer i.e. to `confirm_token_transfer` (i.e. we might want to display the ticker there too).